Firmware/Media/Icons/delete_icon.h

```diff
@@ -1,16 +1,16 @@
 #include <stdint.h>
 
-static const struct
-{
-	    uint16_t width;
-	    uint16_t height;
-		  uint8_t delete_data[24 * 24 + 1];
-} deleteB = {
-	    24, 24,
-	    {   0x00, 0x7E, 0x00, 0x00, 0xEB, 0x00, 0x80, 0x81, 0x01, 0xF0, 0xFF, 0x0F, 
-          0xF0, 0xFF, 0x0F, 0xB0, 0xBD, 0x0E, 0x00, 0x00, 0x00, 0xF0, 0xFF, 0x0F, 
-          0xF0, 0xFF, 0x0F, 0xE0, 0x7E, 0x0F, 0xF0, 0x64, 0x0F, 0xE0, 0x26, 0x07, 
-          0xF0, 0x7E, 0x0F, 0xE0, 0x24, 0x07, 0xE0, 0x24, 0x07, 0xE0, 0xAE, 0x07, 
-          0xE0, 0x74, 0x07, 0xE0, 0x25, 0x07, 0xE0, 0x24, 0x07, 0xE0, 0xB5, 0x07, 
-          0xE0, 0x24, 0x07, 0xE0, 0xFF, 0x07, 0xC0, 0xFF, 0x07, 0xE0, 0xFF, 0x03}
-};
+#include "../../UI/Widgets/Icon.h"
+ 
+static const struct  delete_icon : public Icon {
```

Comment:
There are 2 blanks between struct and name, please check why.

Reply:
Maybe makefile is doing this somehow

Reply:
You should be in full control of the build and not Makefile. Please investigate where this behavior is coming from.

Reply:
Using `$(1)` in `generate_header_file` adds a space in front, I'll try to remove it then

Reply:
Solved!